packages/client-sdk-nodejs/src/batchutils/batch-functions.ts

```diff
@@ -0,0 +1,145 @@
+import {CacheDelete, CacheGet, CacheSet} from '@gomomento/sdk-core';
+import {CacheClient} from '../cache-client';
+import {
+  BatchDeleteRequest,
+  BatchDeleteResponse,
+  BatchGetRequest,
+  BatchGetResponse,
+  BatchSetItem,
+  BatchSetRequest,
+  BatchSetResponse,
+  defaultMaxConcurrentRequests,
+} from './batch-props';
+import {range} from '@gomomento/sdk-core/dist/src/internal/utils';
+
+export {
+  BatchDeleteRequest,
+  BatchDeleteResponse,
+  BatchGetRequest,
+  BatchGetResponse,
+  BatchSetItem,
+  BatchSetRequest,
+  BatchSetResponse,
+  defaultMaxConcurrentRequests,
+} from './batch-props';
```

Comment:
I'm not sure you need `from './batch-props'` here since you've already imported these above. I don't think it hurts anything but may be superfluous strictly speaking.